src/ServiceControl/App.config

```diff
@@ -34,18 +34,18 @@ These settings are only here so that we can debug ServiceControl while developin
     <!--<add name="NServiceBus/Transport" connectionString="Region=;QueueNamePrefix=;TopicNamePrefix=;AccessKeyId=;SecretAccessKey=;S3BucketForLargeMessages=;S3KeyPrefix=" />-->
 
     <!--AzureStorageQueue -->
-    <!--<add name="DefaultEndpointsProtocol=[http|https];AccountName=;AccountKey=;Subscriptions Table=tablename" />-->
+    <!--<add name="NServiceBus/Transport" connectionString="DefaultEndpointsProtocol=[http|https];AccountName=;AccountKey=;Subscriptions Table=tablename" />-->
 
     <!--MSMQ -->
-    <!--<add name="NServiceBus/Transport" connectionString="" />-->
+    <add name="NServiceBus/Transport" connectionString="" />
```

Comment:
Learning is already uncommented above though?
```suggestion
    <!--<add name="NServiceBus/Transport" connectionString="" />-->
```

Reply:
Yep, must have missed re-commenting that one out. Fixed.